Signal/src/network/GiphyAPI.swift

```diff
@@ -89,8 +104,9 @@ enum GiphyFormat {
 
     // TODO: We may need to tweak these constants.
     let kMaxDimension = UInt(618)
-    let kMinDimension = UInt(101)
-    let kMaxFileSize = UInt(3 * 1024 * 1024)
+    let kMinDimension = UInt(60)
```

Comment:
I think we'll want a min dimension of 101 for the sending rendition.

Reply:
Done. I've added a rendition type enum: `.stillPreview, .animatedLowQuality, .animatedHighQuality`